cms/cms_toolbars.py

```diff
@@ -576,11 +570,6 @@ def add_page_menu(self):
             # first break
             current_page_menu.add_break(PAGE_MENU_FIRST_BREAK)
 
-            # page edit
-            with force_language(self.current_lang):
```

Comment:
@Aiky30 I have deleted the `Page -> Edit` menu entry here. 

Largely, I did this because **with versioning it was a different action compared to the `Edit` button** (which potentially creates a new draft). So either versioning needs to replace the `Page -> Edit` menu entry (just like the language menu) to ensure both trigger the same action. Due to its limited usability (in the middle of a menu as opposed to the 1-click-button) I figured it is not worth the effort to replace it in versioning.

Frankly, I still am of this opinion, but had not put forward the reasons. In the end I am open to keeping it but I doubt it is of any use.

Reply:
I forgot one more argument: Since the toolbar now supports all content types they all should make similar use of menus etc.: All content-type specific actions go into a menu (Page settings, Alias settings etc.) Edit/Preview/Revert etc. are buttons in the toolbar. Alias, e.g., does **not** have an Edit option in its menu.

Reply:
I'm in favour of edit being part of the toolbar, not the menu.

That's basically the only way I ever go to edit a page. Same can be said for everyone I've worked with.

Reply:
If another package (djangocms-versioning) in this case, needs to make changes, we can provide the functionality as default int he cms_config, and it can be overridden by other packages. We have to be careful that any existing functionality, although we may not personally use it, should still continue to work where possible.

Reply:
I don't want to get hung up about this. Perhaps this is a different discussion. I reintroduced the `Page->Edit` menu entry.